pkg/services/sqlstore/active_node.go

```diff
@@ -218,3 +229,63 @@ func GetNodeProcessingMissingAlerts(cmd *m.GetNodeProcessingMissingAlertsCommand
 	cmd.Result = &retNode
 	return nil
 }
+
+func ClusteringCleanupCheck(cmd *m.ClusteringCleanupCheckCommand) error {
+	sqlog.Debug("ClusteringCleanupCheck called")
+	lasthb := cmd.LastHeartbeat
+	results, err1 := x.Query(lastCleanupCheckSQL, lasthb-int64(setting.ClusteringCleanupPeriod), lasthb)
+	if err1 != nil {
+		sqlog.Warn("Cleanup check failed", "error", err1)
+		cmd.Result = false
+		return err1
+	}
+	if len(results) > 0 {
+		sqlog.Debug("Cheanup is already done")
```

Comment:
This debug message is confusing. It should be 'Cleanup is complete'. Cleanup already done suggests that cleanup was done by some other node.